src/components/common/DropdownComponent.jsx

```diff
@@ -5,7 +5,6 @@ import PropTypes from 'prop-types';
 const DropdownComponent = props => (
   <Dropdown
     id={props.id}
-    style={{ width: '65%' }}
```

Comment:
This deletion won't affect anything, right?

Reply:
No, it won't. I added the styling for it in the `sharedStyling.scss` file